crt_portal/cts_forms/signals.py

```diff
@@ -25,6 +25,9 @@ def get_client_ip(request):
         ip = request.META.get('REMOTE_ADDR')
     return ip
 
+def get_user_section():
+    current_request = CrequestMiddleware.get_request()
+    return current_request.user.profile.section
```

Comment:
re: the failing test - you need to ensure `hasattr(user, 'profile')` to check whether the user has a profile set up.